src/API/Reports/PerformanceIndicators/Controller.php

```diff
@@ -177,17 +184,43 @@ public function get_analytics_report_data() {
 		}
 	}
 
+	/**
+	 * Get active Jetpack modules.
+	 *
+	 * @return array List of active Jetpack module slugs.
+	 */
+	public function get_active_jetpack_modules() {
+		if ( is_null( $this->active_jetpack_modules ) ) {
+			if ( class_exists( '\Jetpack' ) ) {
```

Comment:
When perusing similar approaches via the wonders of opengrok :) I noticed quite a few places doing a `function_exists()` call on `active_jetpack_modules()` too. Not sure if this was due to the method being added at some point, but perhaps it would be safe to check here too just in case.

Reply:
Done in 4b1289ef.